setup.py

```diff
@@ -1,7 +1,11 @@
+import os
 from setuptools import setup, Extension, find_packages
 
 version="0.0.4"
 
+requirementstxt = os.path.join(os.path.dirname(__file__), 'requirements.txt')
```

Comment:
Is it needed to specify the path? Based on how packages (and this one, too) are usually structured, I think it would be enough to use `"requirements.txt"` at row 7, instead of creating a new variable.